@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -733,15 +729,15 @@ export class SolidLineMaterial extends THREE.RawShaderMaterial
         setShaderMaterialDefine(this, "USE_FADING", value > 0.0);
     }
 
-    get displacementMap(): THREE.Texture | undefined {
+    get displacementMap(): THREE.Texture | null {
```

Comment:
This will never return null, we need to probably change c-tor section:
```
displacementMap: new THREE.Uniform(
                        displacementMap !== undefined ? displacementMap : new THREE.Texture()
                    ),
```

Reply:
That c-tor section doesn't need to change, the displacementMap checked there comes from the DisplacementParameters interface, which has the type "DisplacementMap|undefined".

The getter will return null if someone sets a null displacementMap with the setter.